Dev container (#2735)
maplibre/maplibre-gl-js: CONTRIBUTING.md

## Preparing your Development Environment

### CodeSpaces

By creating a code spcace you should be able to start working immidiately after the post create script finishes running.

> This script basically installes everything written here in the linux part.

Reviewer: ```suggestion
This script basically installs everything written here for Linux.
```
